src/Apcu/bootstrap.php

```diff
@@ -11,10 +11,14 @@
 
 if (!extension_loaded('apc') && !extension_loaded('apcu')) {
     return;
-}
-
-if (!function_exists('apcu_add')) {
+} else if (extension_loaded ('Zend Data Cache')) {
+    // Zend Data Cache refuses to load in the CLI, regardless of the configuration
+    if ('cli' != PHP_SAPI && false !== ini_get('zend_datacache.apc_compatibility')) {
```

Comment:
I'm aware of that limitation.  Unfortunately, Zend Data Cache does the checking internally, and does not have the option that APC does to force enabling in the cli.

The point was to avoid adding the compatibility to the cli, since it wouldn't work anyway.  It will fail gracefully if that check fails - the apc functions will simply return false, the same as any other failure.